packages/web-ui/src/BaseCheckbox/BaseCheckbox.props.ts

```diff
@@ -0,0 +1,25 @@
+import type { ReactNode } from 'react';
+import type { CheckboxProps as RadixCheckboxProps } from '@radix-ui/react-checkbox';
+
+export interface BaseCheckboxProps
+  extends Omit<
+    RadixCheckboxProps,
+    'asChild' | 'value' | 'onCheckedChange' | 'defaultChecked' | 'defaultValue'
+  > {
+  /** The value given as data when submitted with a `name`. */
+  value?: string;
+  defaultValue?: string;
+  /**
+   * The label for the Checkbox. If not using please properly associate the
+   * checkbox with a label using the `aria-label` or `aria-labelledby` props.
+   */
+  label?: ReactNode;
+  /** Helper text for the Checkbox. Will not display if the checkbox group has `helperText` set. */
+  helperText?: ReactNode;
+  /** The controlled checked state of the checkbox. Must be used in conjunction with onCheckedChange. */
+  checked?: boolean;
+  /** Event handler called when the checked state of the checkbox changes. */
+  onCheckedChange?: (checked: boolean) => void;
```

Comment:
Why not just `onChange`?

Reply:
This comes from Radix, and I believe it's because it's a custom implementation, different from the native `onchange` event.